test/obfuscated.spec.js

```diff
@@ -12,84 +11,87 @@ import { getWarningKind } from "./utils/index.js";
 // CONSTANTS
 const FIXTURE_URL = new URL("fixtures/obfuscated/", import.meta.url);
 
-test("should detect 'jsfuck' obfuscation", (tape) => {
+test("should detect 'jsfuck' obfuscation", () => {
   const trycatch = readFileSync(new URL("jsfuck.js", FIXTURE_URL), "utf-8");
   const { warnings } = runASTAnalysis(trycatch);
 
-  tape.strictEqual(warnings.length, 1);
-  tape.deepEqual(getWarningKind(warnings), ["obfuscated-code"].sort());
-  tape.strictEqual(warnings[0].value, "jsfuck");
-  tape.end();
+  assert.strictEqual(warnings.length, 1);
+  assert.deepEqual(getWarningKind(warnings), ["obfuscated-code"].sort());
+  assert.strictEqual(warnings[0].value, "jsfuck");
 });
 
-// test("should detect 'morse' obfuscation", (tape) => {
+// test("should detect 'morse' obfuscation", () => {
```

Comment:
Should we delete or uncomment this test ?

Reply:
https://github.com/NodeSecure/js-x-ray/issues/144 (out the scope of this PR)